### DISCUSSION TITLE
🚀 FR: setting for default value for the Entry Manager > 'Search Titles Only' checkbox

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#4403] [Feature Requests]
### Suggest an idea for ExpressionEngine

Version 7.4.6 made the default for the Entry Manager > 'Search Titles Only' checkbox set to 'Titles only' (via #3924) and whilst I agree it makes the most sense for a default, there's no way to change that for clients that specifically _don't_ want it that way.

The ideal solution would be to make that part of the customizable view on a per-user basis (along with being able to customize the 'columns' in view).   The next best option IMO would be a system-wide CMS option or at the very least a config override to be able to default that checkbox to be **not** checked.

**Comment by robinsowell** [comment]
It looks like the options on where to search and the default have changed over time.  Which I do think lends support to the idea this needs to be more configurable.  I can see doing it in a layout- which would allow it to differ by member role, or by user, or at simplest by system config.  But I think system config is a bit broad if we're hashing out the best option.

**Comment by intoeetive** [comment]
I think the most obvious solution is that we just would make the browser remember last setting (whether it was checked or not). 
@Yulyaswan I believe that's something we can do with the help of localStorage?

**Comment by pyrobob83** [comment]
@intoeetive That's a great solution, even if the setting is remembered globally vs each individual channel. That's solve our issue for sure - thanks for considering this!

**Comment by Yulyaswan** [comment]
PR - https://github.com/ExpressionEngine/ExpressionEngine/pull/4480
